src/internal/HtmlColorParse.cpp

```diff
@@ -0,0 +1,54 @@
+/*-------------------------------------------------------------------------
+HtmlColor::Parse() implementation
+
+Written by Unai Uribarri
+
+I invest time and resources providing this open source code,
+please support me by dontating (see https://github.com/Makuna/NeoPixelBus)
+
+-------------------------------------------------------------------------
+This file is part of the Makuna/NeoPixelBus library.
+
+NeoPixelBus is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as
+published by the Free Software Foundation, either version 3 of
+the License, or (at your option) any later version.
+
+NeoPixelBus is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public
+License along with NeoPixel.  If not, see
+<http://www.gnu.org/licenses/>.
+-------------------------------------------------------------------------*/
+#include "HtmlColor.h"
+
+bool HtmlColor::Parse(const char* name)
+{
+    if (name[0] == '#')
+    {
+        char *err;
```

Comment:
Yes, but don't make the work of latter require the user to follow a standard that doesn't fit how they solved the syntax problem.  This parse should be done as low level as possible; maybe even requiring them to do more to fit the true over the wire protocol; as it maybe used outside of what you plan.  
I guess I am getting at, don't think of this as a Web API, its not, its a string API helper.